pkg/arvo/app/chat-pull-hook.hoon

```diff
@@ -0,0 +1,260 @@
+/-  *invite-store, hook=chat-pull-hook
+/+  default-agent, verb, dbug, store=chat-store, lib=chat-hooks
+|%
++$  card  card:agent:gall
++$  versioned-state
+  $%  [%0 state-0]
+  ==
++$  state-0
+  $:  tracking=(map path ship)
+  ==
+--
+::
+=|  [%0 state-0]
+=*  state  -
+::
+%-  agent:dbug
+%+  verb  |
+::
+^-  agent:gall
+|_  bol=bowl:gall
++*  this  .
+    def  ~(. (default-agent this %|) bol)
+::
+++  on-init
+  ^-  (quip card _this)
+  :_  this
+  =/  inv  [our.bol %invite-store]
+  :~  [%pass / %agent inv %poke %invite-action !>([%create /chat])]
+      [%pass /invites %agent inv %watch /invitatory/chat]
+  ==
+::
+++  on-save  !>(state)
+++  on-load
+  |=  vax=vase
+  =/  old  !<(versioned-state vax)
+  ?-  -.old
+    %0  [~ this(state old)]
+  ==
+::
+++  on-poke
+  |=  [=mark =vase]
+  ^-  (quip card _this)
+  ?.  ?=(%chat-pull-hook-action mark)
+    (on-poke:def mark vase)
+  =/  act  !<(action:hook vase)
+  ?-  -.act
+      %add
+    ?>  (team:title our.bol src.bol)
```

Comment:
Please move this prior to the `?-` as it should apply to all conditions.

Reply:
I would like that behavior to change, yes. No need for the host to be able to send a remove.